config/application.rb

```diff
@@ -19,5 +19,8 @@ class Application < Rails::Application
     config.api_only = true
     config.active_job.queue_adapter = :sidekiq
     config.test_framework = :rspec
+    Rails.autoloaders.main.ignore(Rails.root.join('app/active_admin'))
```

Comment:
Can you please explain those changes

Reply:
It will need when merge activeadmin gem

Reply:
I mean about of all `Rails.autoloaders.main.ignore`